elliottlib/imagecfg.py

```diff
@@ -23,32 +25,26 @@ def base_only(self):
         """
         return self.config.base_only
 
-    def get_latest_build_info(self, product_version):
+    def get_latest_build_info(self, product_version_dict, tag_set, latest_builds, brew_session):
```

Comment:
This is an optional suggestion:
Those arguments for caching purpose expose the implementation to outside of the function and make this function hard to reuse.
Changing the signature of function to something like `def get_latest_build_infos(self, tag): (component name, version, release)`, and cache the results on the caller side.